infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftProject.java

```diff
@@ -48,12 +49,13 @@ public OpenShiftProject(OpenShiftClientFactory clientFactory, String name, Strin
       throws InfrastructureException {
     super(clientFactory, name, workspaceId, false);
     this.routes = new OpenShiftRoutes(name, workspaceId, clientFactory);
-    doPrepare(name, clientFactory.create());
+    doPrepare(name, clientFactory.create(), clientFactory.createOC());
```

Comment:
Should `clientFactory.create()` be invoked here with `workspaceId`?

Reply:
you're right, it should be invoked with the `workspaceId` it seems

Reply:
Done in commit https://github.com/eclipse/che/pull/8805/commits/112e995392f1066e94d6708f5cf2761e38b312eb